make setup-sandbox fails on Windows with error code 255

### Problem summary

 make setup-sandbox fails on Windows with error 255

### Expected behavior

When I execute this command `make setup-sandbox`, I can pull the image successfully.

### Actual behavior

don't pull image, have error 255

### Operating system

Windows

### Platform details

_No response_

### Python version

_No response_

### Node.js version

_No response_

### pnpm version

_No response_

### uv version

_No response_

### How are you running DeerFlow?

Docker (make docker-dev)

### Reproduction steps

1. make check
2. make install
3. make setup-sandbox

### Relevant logs

```shell
If I execute the `make setup-sandbox` command on the Windows platform, the following error message will appear.

"=========================================="
"  Pre-pulling Sandbox Container Image"
"=========================================="
""
make: *** [setup-sandbox] 错误 255
```

### Git state

_No response_

### Additional context

_No response_

--- a/Makefile
+++ b/Makefile
@@ -89,36 +89,7 @@ install:
 
 # Pre-pull sandbox Docker image (optional but recommended)
 setup-sandbox:
-	@echo "=========================================="
-	@echo "  Pre-pulling Sandbox Container Image"
-	@echo "=========================================="
-	@echo ""
-	@IMAGE=$$(grep -A 20 "# sandbox:" config.yaml 2>/dev/null | grep "image:" | awk '{print $$2}' | head -1); \
-	if [ -z "$$IMAGE" ]; then \
-		IMAGE="enterprise-public-cn-beijing.cr.volces.com/vefaas-public/all-in-one-sandbox:latest"; \
-		echo "Using default image: $$IMAGE"; \
-	else \
-		echo "Using configured image: $$IMAGE"; \
-	fi; \
-	echo ""; \
-	if command -v container >/dev/null 2>&1 && [ "$$(uname)" = "Darwin" ]; then \
-		echo "Detected Apple Container on macOS, pulling image..."; \
-		container image pull "$$IMAGE" || echo "⚠ Apple Container pull failed, will try Docker"; \
-	fi; \
-	if command -v docker >/dev/null 2>&1; then \
-		echo "Pulling image using Docker..."; \
-		if docker pull "$$IMAGE"; then \
-			echo ""; \
-			echo "✓ Sandbox image pulled successfully"; \
-		else \
-			echo ""; \
-			echo "⚠ Failed to pull sandbox image (this is OK for local sandbox mode)"; \
-		fi; \
-	else \
-		echo "✗ Neither Docker nor Apple Container is available"; \
-		echo "  Please install Docker: https://docs.docker.com/get-docker/"; \
-		exit 1; \
-	fi
+	@$(RUN_WITH_GIT_BASH) ./scripts/setup-sandbox.sh
 
 # Start all services in development mode (with hot-reloading)
 dev:

--- a/scripts/setup-sandbox.sh
+++ b/scripts/setup-sandbox.sh
@@ -1,0 +1,45 @@
+#!/usr/bin/env bash
+# Pre-pull sandbox container image for DeerFlow
+
+set -uo pipefail
+
+echo "=========================================="
+echo "  Pre-pulling Sandbox Container Image"
+echo "=========================================="
+echo ""
+
+# Try to extract image from config.yaml (handles both commented and uncommented sandbox sections)
+IMAGE=""
+if [ -f "config.yaml" ]; then
+    # Look for uncommented image: field under the sandbox section
+    IMAGE=$(grep -A 20 "^sandbox:" config.yaml 2>/dev/null | grep "^  image:" | awk '{print $2}' | head -1 || true)
+fi
+
+if [ -z "$IMAGE" ]; then
+    IMAGE="enterprise-public-cn-beijing.cr.volces.com/vefaas-public/all-in-one-sandbox:latest"
+    echo "Using default image: $IMAGE"
+else
+    echo "Using configured image: $IMAGE"
+fi
+
+echo ""
+
+if command -v container >/dev/null 2>&1 && [ "$(uname)" = "Darwin" ]; then
+    echo "Detected Apple Container on macOS, pulling image..."
+    container image pull "$IMAGE" || echo "⚠ Apple Container pull failed, will try Docker"
+fi
+
+if command -v docker >/dev/null 2>&1; then
+    echo "Pulling image using Docker..."
+    if docker pull "$IMAGE"; then
+        echo ""
+        echo "✓ Sandbox image pulled successfully"
+    else
+        echo ""
+        echo "⚠ Failed to pull sandbox image (this is OK for local sandbox mode)"
+    fi
+else
+    echo "✗ Neither Docker nor Apple Container is available"
+    echo "  Please install Docker: https://docs.docker.com/get-docker/"
+    exit 1
+fi

